mythril/laser/ethereum/plugins/implementations/coverage.py

```diff
@@ -47,13 +54,36 @@ def execute_state_hook(global_state: GlobalState):
             # Record coverage
             code = global_state.environment.code.bytecode
 
-            if code not in coverage.keys():
+            if code not in self.coverage.keys():
                 number_of_instructions = len(
                     global_state.environment.code.instruction_list
                 )
-                coverage[code] = (
+                self.coverage[code] = (
                     number_of_instructions,
                     [False] * number_of_instructions,
                 )
 
-            coverage[code][1][global_state.mstate.pc] = True
+            self.coverage[code][1][global_state.mstate.pc] = True
+
+        @symbolic_vm.laser_hook("start_sym_trans")
+        def execute_start_sym_trans_hook():
+            self.initial_coverage = self._get_covered_instructions()
+
+        @symbolic_vm.laser_hook("stop_sym_trans")
+        def execute_stop_sym_trans_hook():
+            end_coverage = self._get_covered_instructions()
+            log.info(
+                "Number of new instructions covered in tx %d: %d"
+                % (self.tx_id, end_coverage - self.initial_coverage)
+            )
+            self.tx_id += 1
+
+    def _get_covered_instructions(self) -> int:
```

Comment:
When will this function get executed?

Reply:
In the two previous hooks, whom are executed at each transaction end & start